docs/api/datasets/bikeshare.rst

```diff
@@ -0,0 +1,32 @@
+.. -*- mode: rst -*-
```

Comment:
This is an example of a regression dataset detail page. This information is taken from the dataset README. The question is if we want to include this much information, add information about the meta (e.g. column names, class names, etc), add examples, etc. What would be most effective for these docs?

Reply:
Since all of the information about the dataset is available on the UCI Machine Learning Repository website, I think providing a link to the dataset README would be sufficient.  

Also, examples would be nice, but I don’t think it is necessary.  We have plenty of examples in other places.

I will leave it to you to make the final decision on how to approach this.